syncstorage-spanner/src/models.rs

```diff
@@ -1417,8 +1412,8 @@ impl SpannerDb {
 
         let mut ids = vec![];
         let mut modifieds = vec![];
-        while let Some(row) = stream.next_async().await {
-            let mut row = row?;
+        while let Some(row) = stream.try_next().await? {
+            let mut row = row;
```

Comment:
there's a couple more of these here and below
```suggestion
        while let Some(mut row) = stream.try_next().await? {
```